.github/workflows/test_live_variant_on_custom_liveview_project.yml

```diff
@@ -0,0 +1,69 @@
+name: "Test Live variant on custom LiveView project"
```

Comment:
Having `live` both seems redundant. The following would be sufficient and would be consistent with the other workflows:

```suggestion
name: "Test variant on custom LiveView project"
```

Reply:
It follows the API variant @olivierobert :) 

![image](https://user-images.githubusercontent.com/11751745/100995862-efced280-358a-11eb-8e36-bbd8b2e6b54f.png)

![image](https://user-images.githubusercontent.com/11751745/100995876-f52c1d00-358a-11eb-9459-117dd2573766.png)